schemas/context/opportunity.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/opportunity",
```

Comment:
In order to accommodate the design of the Activity Log, the contactmap has to be an independent entity. Instead of denormalizing the opportunity and the opportunity contact role, we need to design them separately. ActivityLog registers on a single entity tuple record at a time hence it is not feasible to have it in the denormalized state. 

The opportunityContactRole will have a technicalPK, opportunityID and contactID plus other attributes to have a star like shape to the ActivityLog.